components/page.js

```diff
@@ -116,6 +116,22 @@ export default ({ hackathons, currentPath, children }) => {
               Sponsors
             </NavItem>
           </Link>
+          <Link href="/[id]/FeatureFlags" as={`/${currentPath}/FeatureFlags`}>
+            <NavItem
+              onClick={() => {
+                if (!window.location.href.includes('FeatureFlags')) {
+                  setTimeOut(
+                    setTimeout(() => {
+                      setLoading(true);
+                    }, 750)
+                  );
+                }
+              }}
```

Comment:
Maybe we could reduce this to a helper?

Reply:
hm maybe this change is better in your Link PR? we can merge that one in first and I'll copy the helper for this branch.

Reply:
soundsgoodtome